packages/cozy-stack-client/src/SettingsCollection.js

```diff
@@ -50,10 +64,7 @@ class SettingsCollection extends DocumentCollection {
 
     const resp = await this.stackClient.fetchJSON('GET', `/settings/${path}`)
     return {
-      data: DocumentCollection.normalizeDoctypeJsonApi(SETTINGS_DOCTYPE)(
-        { id: `/settings/${path}`, ...resp.data },
-        resp
-      )
+      data: normalizeSettings({ id: `/settings/${path}`, ...resp.data })
```

Comment:
I feel like we should improve [normalizeDoctypeJsonApi](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/DocumentCollection.js#L85) to deal with the `attributes` rather than having to re-implement this normalize function for almost every collection. @Crash-- what do you think?

Reply:
The idea was to have a breaking change with a limited impact so that we could move in gradually. But I agree with your view too